modules/ROOT/pages/managing-schedules.adoc

```diff
@@ -29,11 +29,12 @@ You can trigger the job while it's running without updating the schedule in your
 When using the *Schedules* tab, consider the following:
 
 * CloudHub manages the Schedulers and triggers them in only one worker, even if more are available.
-* All schedules are based in the UTC timezone, regardless of the geographic region where your 
+* All schedules are based in the UTC timezone, regardless of the geographic region where your
 application is deployed.
 * The minimum recommended frequency is 10 seconds between calls.
 * If a scheduled job is not triggered because the application is not running,
 CloudHub triggers the job as soon as the application restarts.
+* When schedule is used to trigger a flow with subflow, it waits until both main flow and subflow complete before scheduling next run.
```

Comment:
@tonylsw - can you change this to:
When a schedule is used to trigger a flow with subflow, it waits until both main flow and subflow complete before scheduling the next run.

I would, but then I can't approve it (SoD) :(

Reply:
updated